### PR TITLE
Feat/rmesh

### PR DESCRIPTION
**Pull request by Dev43**

### CODE REVIEW

File: tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "address": "0x8d5682941ce456900b12d47ac06a88b47c764ce1",
+    "decimals": 18,
+    "name": "RightMesh Token",
+    "ens_address": "",
+    "website": "https://www.rightmesh.io/",
+    "logo": {
+      "src": "https://avatars2.githubusercontent.com/u/29805586?s=128",
+      "width": "92",
+      "height": "92",
+      "ipfs_hash": "QmQ9tzSaUpjrhsZ6vFjp8P5FcBWFEKrYD8joi6d8HVXcdc"
```
Comment (compscidr) [inline tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json]: Dunno if im just dumb at ipfs or what - but I can't seem to get this hash to resolve.

File: tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "support": {
+      "email": "contact@rightmesh.io"
```
Comment (compscidr) [inline tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json]: not sure if contact@rightmesh.io goes anywhere - I emailed it today to see if anyone answered and got nothing back.

I do know that tech@rightmesh.io will go to me and the dev team so maybe use that.

File: tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "social": {
+      "blog": "https://medium.com/rightmesh",
+      "chat": "",
```
Comment (compscidr) [inline tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json]: could put rocketchat  in here? https://rightmesh.rocket.chat/

**Review by compscidr — APPROVED** [review]
Logos look fine, just a couple of minor comments.

File: tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json (hunk 1 above)
Comment (Dev43) [reply tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json]: Sorry about that, I forgot to add it to my server (my local computer wasn't connected when you checked it). I've now put it on a server that is reliable and is supposed to work 24/7 but I would recommend having at least 1 IPFS node that hosts the same file on your end (doing a simple `ipfs get  QmQ9tzSaUpjrhsZ6vFjp8P5FcBWFEKrYD8joi6d8HVXcdc` will make it so your own server also hosts it and adds it to the DHT

Comment (compscidr) [reply tokens/eth/0x8d5682941ce456900b12d47ac06a88b47c764ce1.json]: Perfect - yep have an ipfs node running permanently on my miner at home, so I'll do the get there. Tried on my local machine and have it now as well - looks good.

**Comment by Dev43** [comment]
@compscidr  Please check again before final merge

File: tokens/eth/0x8D5682941cE456900b12d47ac06a88b47C764CE1.json [hunk 4]
```diff
@@ -1,34 +1,32 @@
...
+    "instagram": "https://www.instagram.com/rightmesh/",
+    "linkedin": "https://www.linkedin.com/company/right-mesh/",
+    "reddit": "https://www.reddit.com/r/RightMesh/",
     "slack": "",
-    "telegram": "",
-    "twitter": "",
-    "youtube": ""
+    "telegram": "https://t.me/RightMesh_Official",
+    "twitter": "https://twitter.com/Right_Mesh",
+    "youtube": "https://www.youtube.com/channel/UCa-am0tFMgJbDP2_yYFhfRA"
```
Comment (compscidr) [inline tokens/eth/0x8D5682941cE456900b12d47ac06a88b47C764CE1.json]: Can change this to https://www.youtube.com/rightmesh

**Comment by compscidr** [comment]
Just getting someone to double check that all of our social links are the official ones and that the official token address is correct. I think they are having the Halloween party currently so they might not answer super fast. If you get tired of waiting you can prob just submit. I did a quick check and they seemed fine to me.

**Comment by Dev43** [comment]
@compscidr all good. Just let me know when you'd like me to pull the trigger!

**Comment by Dev43** [comment]
@aldrindsouza  all good changed it last night. Please do a final check and i'll get this merged in the original repo!

**Comment by aldrindsouza** [comment]
@Dev43 social links and token address checked-  looks good to go.

**Pull request merged**